pom.xml

```diff
@@ -209,6 +210,9 @@
                         <parallel>suites</parallel>
                         <useUnlimitedThreads>true</useUnlimitedThreads>
                         <forkMode>always</forkMode>
+                        <excludes>
+                            <exclude>${someModule.test.excludes}</exclude>
```

Comment:
Why this?

Reply:
i'd like to excute unit tests, and this config enable to exclude some tests which need other environment like tidb.
You can find the relevant commandLine in verify.yml. Unfortunately i did't found a way to exclude some tests in commandLine without this config

Reply:
It's going to be expanded to value of system property `someModule.test.excludes`, therefore could control what tests to skip from command line options when starting JVM.